meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -133,6 +133,12 @@ if test "${fiovb.is_secondary_boot}" = "0"; then
 		imx_secondary_boot 0
 		reset
 	fi
+else
+	# Save fiovb.is_secondary_boot state for allowing userspace
+	# to easily identify the boot mode via environment
+	if test -z "${fiovb_rpmb}"; then
+		run saveenv_mmc
+	fi
```

Comment:
At this stage you will save all the uboot_env variable and the fiovb.... versions of the same variables is that what you want?

Reply:
That is happening anyway on every safe.